README.md

```diff
@@ -48,6 +48,7 @@ Check out our getting started tutorial on launching your first Kubernetes cluste
   * [Backup Kubernetes resources](/Documentation/kubernetes-on-aws-backup.md)
   * [Restore Kubernetes resources](/contrib/cluster-backup/README.md)
   * [Journald logging to AWS CloudWatch](/Documentation/kubernetes-on-aws-journald-cloudwatch-logs.md)
+    * [kube-aws up/update feedback](/Documentation/kubernetes-on-aws-journald-cloudwatch-logs.md)
```

Comment:
Seems like it is `#kube-aws-upupdate-feedback` rather than `#kube-aws-up-update-feedback` - the hyphen between `up` and `update` dropped over my expectation.
![image](https://user-images.githubusercontent.com/22009/28097558-d267441e-66eb-11e7-9bfd-2789b6b9278d.png)